mod_reforged/rf_sling_pot_skill.nut

```diff
@@ -0,0 +1,98 @@
+// This class is combines common properties of the new sling-an-item skills
+::Reforged.applySlingPotSkill <- function( o )
+{
+	::logWarning("create is called");
+	o.m.ID = ::MSU.String.replace(o.m.ID, "throw", "sling");
+	o.m.Name = ::MSU.String.replace(o.m.Name, "Throw", "Sling");
+	o.m.Description = ::MSU.String.replace(::MSU.String.replace(o.m.Description, "throw", "sling"), "Throw", "Sling");
+	o.m.Order = ::Const.SkillOrder.UtilityTargeted + 1000;
+	o.m.ActionPointCost = 5;	// Flasks have 5
+	o.m.FatigueCost = 25;		// Flasks have 20
+	o.m.MinRange = 2;			// Flasks have 0
+	o.m.MaxRange = 6;			// Flasks have 3
+	o.m.MaxLevelDifference = 4;	// Flasks have 3
+	o.m.IsHidden = true;
+
+	local getTooltip = o.getTooltip;
+	o.getTooltip = function()
+	{
+		local ret = getTooltip();
+		ret.push({
+			id = 8,
+			type = "text",
+			icon = "ui/icons/vision.png",
+			text = "Has a range of " + ::MSU.Text.colorGreen(this.getMaxRange()) + " tiles on even ground, more if shooting downhill"
```

Comment:
Probably wanna use `this.getRangedTooltip()` function provided by MSU. In that case you'll need to add `AdditionalAccuracy` and `AdditionalHitChance` to `o.m`.

Reply:
I'll use the hard-coded variant for now. Adding these dummy variables just for an automated tooltip to appear pus me off.